packages/apollo-link-dedup/src/dedupLink.ts

```diff
@@ -14,7 +16,7 @@ export class DedupLink extends ApolloLink {
     string,
     Observable<FetchResult>
   > = new Map();
-  private subscribers: Map<string, any> = new Map();
+  private subscribers: Map<string, Array<Observer<FetchResult>>> = new Map();
```

Comment:
Why not `Map<string, Set<Observer<FetchResult>>>`?

Reply:
Hi @benjamn ! Glad you're reviewing this. 
That's a good suggestion. I'll revise it.